### PR TITLE
feat(packages): wip

### PR DESCRIPTION
**Pull request by Orta21**
Ref. metriport/metriport-internal#419

Ref:419

### Description

Add cordova to our packages
### Release Plan

_[Document your release plan to minimize downtime]_

- [ ] Merge once approved

### CODE REVIEW

File: docs/devices-api/more-info/cordova.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: "Cordova"
+description: "A plugin to integrate with Metriport on Cordova - including Apple Health integrations."
+---
+
+<Info>
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
```
Comment (Goncharo) [inline docs/devices-api/more-info/cordova.mdx:9]: Nit: extra `"`

File: docs/devices-api/more-info/cordova.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd platforms/ios
+pod install
+```
+
+## Usage
+
+To use our SDK go to your index.html add:
```
Comment (Goncharo) [inline docs/devices-api/more-info/cordova.mdx:27]: nit: To use our SDK, go to your `index.html` and add:

File: docs/devices-api/more-info/cordova.mdx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+```html
+<body>
+  ...
+  <button id="show-widget">Open widget</button>
+  ...
+</body>
+```
+
+Then in your js file add:
```
Comment (Goncharo) [inline docs/devices-api/more-info/cordova.mdx:37]: nit: Then, in your js file add:

File: docs/devices-api/more-info/cordova.mdx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+```sh
+cordova plugin add https://github.com/metriport/cordova-sdk
+```
+
+You'll need to ensure that the MetriportSDK pod is installed as well.
+
+```sh
+cd platforms/ios
+pod install
```
Comment (Goncharo) [inline docs/devices-api/more-info/cordova.mdx:22]: they won't have to `pod install` a specific pod?

**Review by Goncharo — APPROVED** [review]
LET'S FUCKING GO 🚀  - few minor nits, otherwise :shipit:

File: docs/devices-api/more-info/cordova.mdx (hunk 4 above)
Comment (Orta21) [reply docs/devices-api/more-info/cordova.mdx:22]: No it works the same as react native where you install the package (or in this case plugin) and then after you run `pod install` in ios itll just see which pods to download based off of what it installed

File: docs/devices-api/getting-started/connect-widget.mdx [hunk 5]
```diff
@@ -289,29 +287,75 @@ const styles = StyleSheet.create({
...
+    token: "CONNECT_TOKEN",
+  })
+);
 ```
 
 <Tip>
-  Our GitHub repo has [an example implementation](https://github.com/metriport/metriport/tree/develop/examples/react-native) of this in a React Native app.
+  Our GitHub repo has [an example
+  implementation](https://github.com/metriport/metriport/tree/develop/examples/react-native) of this
+  in a React Native app.
```
Comment (leite08) [inline docs/devices-api/getting-started/connect-widget.mdx:358]: Is this really related to Cordova? It seems React Native only, no?

File: docs/devices-api/more-info/cordova.mdx (hunk 1 above)
Comment (leite08) [inline docs/devices-api/more-info/cordova.mdx:9]: NABD, but this text gives the impression this can only be used to connect with AppleHealth, but one could use the Cordova connector for Android/other providers as well, right?

File: docs/devices-api/more-info/cordova.mdx (hunk 2 above)
Comment (leite08) [reply docs/devices-api/more-info/cordova.mdx:27]: I see you marked these as resolved but I still see the original code, maybe forgot to push the changes?

File: docs/devices-api/more-info/react-native.mdx [hunk 6]
```diff
@@ -4,8 +4,9 @@ description: "A library to integrate with Metriport on React Native - including
 ---
 
 <Info>
-  Before getting started, visit the [Apple Health docs](/devices-api/more-info/apple-health#initial-setup) and
-  follow the Initial Setup section to configure HealthKit."
+  Before getting started, visit the [Apple Health
+  docs](/devices-api/more-info/apple-health#1-enable-healthkit) and follow the Enable HealthKit
+  section to configure HealthKit."
```
Comment (leite08) [inline docs/devices-api/more-info/react-native.mdx:9]: Likewise, text focused on AppleHealth but one could be using ReactNative and not connecting to AppleHealth?

File: examples/cordova/package.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
+{
+  "name": "com.metriport.cordova",
+  "displayName": "MetriportTestApp",
+  "version": "1.0.0",
+  "description": "A sample Metriport Cordova application that uses our healthkit plugin.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (leite08) [inline examples/cordova/package.json:8]: nit: maybe just leave this out, since its an example and the test script doesn't add to it?

File: docs/devices-api/more-info/cordova.mdx [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+Then in your js file add:
+
+```js
+document.getElementById("show-widget").addEventListener("click", () =>
+  metriportconnectwidget.show({
+    clientApiKey: "CLIENT_API_KEY",
+    token: "CONNECT_TOKEN",
+  })
+);
+```
```
Comment (leite08) [inline docs/devices-api/more-info/cordova.mdx:46]: NABD, but this is duplicated on `docs/devices-api/getting-started/connect-widget.mdx`, right? Maybe create a snippet and reuse it

**Review by leite08 — APPROVED** [review]
LGTM :shipit: 

Since you're working on it, I thought maybe we could address this, if you guys agree:
- I miss file names for each code section; it helps when I'm following a step-by-step to have all info there, so I don't nee to be wondering about how to make it work - not everyone will be super knowledgeable about this.

image

File: docs/devices-api/more-info/cordova.mdx [hunk 9]
```diff
@@ -24,7 +24,7 @@ pod install
 
 ## Usage
 
-To use our SDK go to your index.html add:
+To use our SDK go to your index.html and add:
```
Comment (leite08) [inline docs/devices-api/more-info/cordova.mdx:27]: nit: wrap index.html around backticks to render as code, and comma after "To use our SDK"
